.travis.yml

```diff
@@ -50,19 +50,19 @@ before_install:
     - go get -u github.com/axw/gocov/gocov
     - go get -u github.com/mattn/goveralls
     - go get -u golang.org/x/tools/cmd/cover
-    - go get -u github.com/fzipp/gocyclo
 
     # Install godep for dependency management
     - go get -u github.com/tools/godep
 
     # Merge all subpackage code coverage reports into a single report
     - go get -u github.com/modocache/gover
 
-    # Install varcheck
-    - go get -u github.com/opennota/check/cmd/varcheck
+    # Get tools used in Makefile
+    - make get-tools
 
-    # Install deadcode
-    - go get -u github.com/mendersoftware/deadcode
+install:
+    # Get dependencies
+    - go get -t ./...
```

Comment:
Indeed, I discovered this confusing behavior as well in #62. But now `go get ./...` is prevented, and everything is supposed to be picked up from `vendor`, aka Go 1.6 style.